tests/PSR7/ServerRequestTest.php

```diff
@@ -58,44 +56,36 @@ public function testItValidatesBodyHasUnexpectedTypeRed() : void
             ->withoutHeader('Content-Type')
             ->withHeader('Content-Type', 'unexpected/content');
 
-        try {
-            $validator = (new ValidatorBuilder())->fromYamlFile($this->apiSpecFile)->getServerRequestValidator();
-            $validator->validate($request);
```

Comment:
This also fixed some tests - some have missing `$this->fail`. Also expecting expection directly is more preferred flow, I think